src/course-unit/data/selectors.js

```diff
@@ -8,5 +8,6 @@ export const getCourseSectionVertical = (state) => state.courseUnit.courseSectio
 export const getCourseId = (state) => state.courseDetail.courseId;
 export const getSequenceId = (state) => state.courseUnit.sequenceId;
 export const getCourseVerticalChildren = (state) => state.courseUnit.courseVerticalChildren;
+export const getCourseVerticalChildren2 = (state) => state.courseUnit.courseVerticalChildren.children;
```

Comment:
Was this just for debug, or where this is used?

Reply:
Good catch! Thanks